source/components/reveal/HxReveal.less

```diff
@@ -1,8 +1,10 @@
+@import "../HxElement";
+
 :host {
   display: block;
 
-  &([open]) > #content {
-    display: block;
+  &([open]) #content {
+    display: contents;
```

Comment:
According to the [CSS Specification](https://drafts.csswg.org/css-scoping/#slots-in-shadow-tree), the `<slot>` element gets a default display property of `display: contents;`. This essentially means that the `<slot>` element should be replaced with its contents (unwrapping its children and removing it from the hierarchy).

By setting ANY other display property, the `<slot>` element would be placed in the hierarchy and CSS inheritance would have to go through the slot before it gets back to the light dom. As such, we're setting its display property back to `contents` to preserve its original behavior.